.travis.yml

```diff
@@ -3,18 +3,12 @@ rust:
 - beta
 - stable
 - nightly
-matrix:
-  include:
-  - rust: nightly-2018-07-22
-    env: CLIPPY=YESPLEASE
-    script:
-    - rustup component add clippy-preview
-    - cargo clippy -- -D warnings
-  - rust: 1.27.2
```

Comment:
The advantage of pinning the version for rustfmt is your CI won't break when a new version of Rust is out that tweaks the format.